project-management/src/main/java/life/qbic/projectmanagement/application/ProjectInformationService.java

```diff
@@ -205,6 +210,35 @@ public void removeFunding(ProjectId projectId) {
   }
 
   public void updateModifiedDate(ProjectId projectID, Instant modifiedOn) {
+    // The update might fail due to an optimistic locking exception (concurrent access of other processes)
+    // To address this, the update is tried until it will eventually not throw the locking exception anymore.
+    // This approach naively assumes, that the locked resource will be released eventually again by the other process.
+    var attempt = 1;
+    while(true) {
+      try {
+        tryToUpdateModifiedDate(projectID, modifiedOn);
+        return;
+      } catch (OptimisticEntityLockException e) {
+        log.debug("Optimistic lock exception occurred while updating modified date for project " + projectID);
+      }
+      try {
+        Thread.sleep(calcBase2Duration(attempt));
+      } catch (InterruptedException e) {
+        log.error("Interrupted while updating modified date for project " + projectID);
+        // We try one last time
+        tryToUpdateModifiedDate(projectID, modifiedOn);
+        Thread.currentThread().interrupt();
+      }
+      attempt++;
+    }
+  }
```

Comment:
As this is only updating the modified date, maybe add a break to the while loop in case the current date is already later then the modifiedOn date?